### DISCUSSION TITLE
Routing in Rsbuild

### DISCUSSION THREAD

**Discussion by sagardwivedi** [web-infra-dev/rsbuild#2913] [General]
How do I set up routing in rsbuild? Can you provide a step-by-step guide on implementing routing in a web application using rsbuild?

**Comment by chenjiahan** [comment]
Rsbuild is a build tool and does not handle client routes. You can use route libs like react-router or vue-router.

For the routing of Rsbuild dev server, you can refer to https://rsbuild.dev/guide/basic/server#page-routing
